### ISSUE TITLE
Not properly working on QNAP

### ISSUE THREAD

**Issue by PopcornMachina** [alexjustesen/speedtest-tracker#302]
I have encountered some problems with the Container Station on a QNAP NAS.
I have loaded the Docker Container and added this volume: /config | /share/Container/speedtesttracker

The Tracker is running in bridge mode with a static IP (QNAP NAS is 192.168.1.11, and the Tracker is accessible on 192.168.1.23)
I can test my speed and see also some results. But:

Bug 1: The time on the machine is different from the server or my PC
Timezone for the container and in the settings: Europe/Berlin showing the test was run on 19:01:37.
My PC shows 19:51

Bug 2: The time is completely wrong
Run a manual test at 19:5 and the dashboard says 19:01:37.
Run a second test at 19:54 and the dashboard says 19:01:47 (WTF? just 10 seconds?)

Bug 3: Crontab is not working.
Set */15 * * * * in the settings but nothing happens. Checked the crontab via /bin/sh:
# crontab -l
MAILTO=""
* * * * * webuser cd /var/www/html && php artisan schedule:run >> /dev/null 2>&1
# 

Edited the crontab in the shell to */5 * * * *

But nothing happens after all. That is really annoying,

Long story short: Only manual tests are working.

**Comment by alexjustesen** [comment]
@PopcornMachina are you setting the `TZ=` when creating the container and/or changing it in the UI under general settings?

**Comment by PopcornMachina** [comment]
I used "Europe/Berlin" for the container while creating it and set the same in the UI settings.
My QNAP is also running with "Europe/Berlin" and get its time from 0.de.pool.ntp.org (Router is master and uses 0.de.pool.ntp.org for broadcasting it to every client).

But I have also encountered some other problems with other containers (like Heimdall advanced features are not working, like showing the last speedtest results or the usage of my nextcloud and so on).
It might be a general problem with QNAP and the Container Station inside. Even Portainer will not work without problems.
I'm switchting to a normal mini PC and will set up Proxmox in my network for further tests (Speedtest tracker on QNAP with link to Heimdall on Proxmox and Speedtest tracker on Proxmox with link to Heimdall).

**Comment by alexjustesen** [comment]
@PopcornMachina QNAP is one of the devices I don't have in my home lab so I can't test it locally. If you're having other issues itmight be a result of QNAP. Going to close this, if the issues come back with another OS feel free to post and we can re-open the issue.
